cEP-0020.md

```diff
@@ -0,0 +1,543 @@
+# Newcomer metrics and Gamification
+
+| Metadata |                                                        |
+| -------- | ------------------------------------------------------ |
+| cEP      | 20                                                     |
+| Version  | 1.0                                                    |
+| Title    | Newcomer metrics and Gamification system               |
+| Authors  | Shrikrishna Singh <mailto:krishnasingh.ss30@gmail.com> |
+| Status   | Proposed                                               |
+| Type     | Process                                                |
+
+## Abstract
+
+This cEP describes how a gamification system will automate the
+process of becoming a `Developer` from a `Newcomer`, as a part of
+this
+[GSoC project](https://summerofcode.withgoogle.com/projects/#5892040252981248).
+
+For the implementation of this process, this project will
+incorporate a gamification system which will allow us to
+track the progress of newcomers and assign some points to
+each of the activities they do. There will be different
+levels based on point thresholds and a newcomer will also
+get some special privileges after the completion of each
```

Comment:
need to be clearer here -- 'get' isnt correct. this project doesnt give the privileges.